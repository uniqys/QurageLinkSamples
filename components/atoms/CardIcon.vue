<template>
  <a href="#" class="card-header-icon" aria-label="more options">
    <span class="icon">
      <i class="fas fa-angle-down" aria-hidden="true" />
    </span>
  </a>
</template>

<script lang="ts">
import { Component, Vue } from 'nuxt-property-decorator'

@Component
export default class CardIcon extends Vue {}
</script>
