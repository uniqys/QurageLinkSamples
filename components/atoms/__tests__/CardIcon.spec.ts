import { shallowMount } from '@vue/test-utils'
import CardIcon from '~/components/atoms/CardIcon.vue'

describe('CardIcon.vue', () => {
  it('should to match snapshot', () => {
    const wrapper = shallowMount(CardIcon)
    expect(wrapper.element).toMatchSnapshot()
  })
})
