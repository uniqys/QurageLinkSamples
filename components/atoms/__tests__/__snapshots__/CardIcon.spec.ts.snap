// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardIcon.vue should to match snapshot 1`] = `
<a
  aria-label="more options"
  class="card-header-icon"
  href="#"
>
  <span
    class="icon"
  >
    <i
      aria-hidden="true"
      class="fas fa-angle-down"
    />
  </span>
</a>
`;
